news/2022-07-20-new-featured-artist-exyl.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "New Featured Artist: Exyl"
+date: 2022-07-20 10:00:00 +0000
+---
+
+@everyone **Exyl** is officially an osu! Featured Artist.
```

Comment:
I'm guessing the `@everyone` is some meme I'm missing?

Reply:
yeah, it's a reference to discord pings (and the fact that two of his songs are named after pings) and the overall heavy presence on the platform the artist has i believe